src/lib.rs

```diff
@@ -746,6 +746,11 @@ impl Config {
                 }
                 _ => {}
             }
+        } else {
+            let cmake_version = cmake_version();
+            if cmake_version.0 >= 3 && cmake_version.1 > 12 {
+                parallel_flags = Some(String::from("--parallel"));
```

Comment:
Is `--parallel` super new? Ideally we wouldn't do a version check since that's an extra process and can take some time, but it's not the end of the world.

Reply:
Also, to clarify, I think this only gets run outside of Cargo's build scripts since Cargo should always set `NUM_JOBS`, but are you using the `cmake` crate in that style?

Reply:
Since cmake 3.12. I use it the cmake crate with 2 other crates of mine. fltk-rs and soloud-rs. For soloud, the c++ code base is small so it builds fast. But with fltk I had noticed slower build times when building on windows-msvc. Manually passing NUM_JOBS didn't seem to help much. So I think it's the m flag thing.